src/Modules/Manage/Dnn.PersonaBar.Users/Components/Prompt/Utilities.cs

```diff
@@ -1,22 +1,18 @@
-﻿using System.Collections.Generic;
-using System.Net;
+﻿using System;
 using Dnn.PersonaBar.Library.Prompt.Models;
 using DotNetNuke.Entities.Portals;
 using DotNetNuke.Entities.Users;
-using DotNetNuke.Services.Localization;
 
 namespace Dnn.PersonaBar.Users.Components.Prompt
 {
     public class Utilities
     {
+        private static readonly IUserValidator _userValidator = new UserValidator();
+
+        [Obsolete("Deprecated in 9.2.1. Use IUserValidator.ValidateUser")]
         public static ConsoleErrorResultModel ValidateUser(int? userId, PortalSettings portalSettings, UserInfo currentUserInfo, out UserInfo userInfo)
         {
-            userInfo = null;
-            if (!userId.HasValue) return new ConsoleErrorResultModel(Localization.GetString("Prompt_NoUserId", Constants.LocalResourcesFile));
-
-            KeyValuePair<HttpStatusCode, string> response;
-            userInfo = UsersController.GetUser(userId.Value, portalSettings, currentUserInfo, out response);
-            return userInfo == null ? new ConsoleErrorResultModel(response.Value) : null;
+            return _userValidator.ValidateUser(userId, portalSettings, currentUserInfo, out userInfo);
```

Comment:
Good, I didn't think of doing that. 👍 
Still, please replace usages of `Utilities` with `UserValidator`